script/crossbinary

```diff
@@ -27,7 +30,7 @@ OS_ARCH_ARG=(386 amd64)
 for OS in ${OS_PLATFORM_ARG[@]}; do
   for ARCH in ${OS_ARCH_ARG[@]}; do
     echo "Building binary for $OS/$ARCH..."
-    GOARCH=$ARCH GOOS=$OS CGO_ENABLED=0 go build -ldflags "-s -w -X github.com/containous/traefik/version.Version=$VERSION -X github.com/containous/traefik/version.Codename=$CODENAME -X github.com/containous/traefik/version.BuildDate=$DATE" -o "dist/traefik_$OS-$ARCH" ./cmd/traefik/
+    /bin/bash -e -c "GOARCH=$ARCH GOOS=$OS CGO_ENABLED=0 $GO_BUILD_CMD -o \"dist/traefik_$OS-$ARCH\" ./cmd/traefik/"
```

Comment:
What's the reason we need to create a sub shell here?

Besides, we cannot use `/bin/bash` since it's not portable.